data/distro.yml

```diff
@@ -405,6 +405,7 @@
         <pre><code>
           <span class="unselectable">#</span> flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
         </code></pre>
+        <p><b>Important note:</b> As of March 2021, Raspberry Pi computers still ship with the 32-bit version of Raspberry Pi OS. However Flathub started phasing out support for that architecture. If you consider Flathub as an important source of applications, it is recommended to use Raspberry Pi OS 64-bit as newer applications are more likely to be available for that platform, while other one more likely to be updated.</p>
```

Comment:
"while other one more likely to be updated" - this part still does not seem to be correct to me.

Reply:
I just removed it.